content/reference/ipaddresses.md

```diff
@@ -0,0 +1,28 @@
+---
+title: IP Addresses
+weight: 199
+---
+
+<!-- vale off -->
+## IP Addresses
+
+Upbound Infrastructure uses the following egress addresses for hosted control
```

Comment:
These are just the MCP egresses? What about from the main cluster for Librarian & the GitHub App?

I think we might need a slightly more detailed paragraph here. I'm running short on time, but I'll make a suggestion later if I have time. But I think that a reader who doesn't have all your context may have trouble understanding.

Reply:
Suggested text:

## Allowing access from Upbound 
Upbound makes outbound connections in three ways:
1. Connections to GitHub to copy the contents of a repository containing your Configuration
2. Connections to GitHub to create your Configurations repository with the Upbound GitHub App. 
3. Connections from Upbound Control Planes to your infrastructure. For example, if you use provider-kubernetes to configure a Kubernetes cluster, there will be a connection to your Kubernetes cluster. 

If you restrict the IP addresses that are allowed to communicate with GitHub or your infrastructure, you'll need to allow the Upbound IP addresses.

To allow access to Github, allow access to the following IP addresses:
* X.X.X.X
* Y.Y.Y.Y

To allow access to your infrastructure from a control plane, allow access to the following IP addresses:
* A.A.A.A
* B.B.B.B